src/v2/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -192,8 +194,9 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
 
     return (
       this.state.shippingOption === "SHIP" &&
-      processWithArtaShipping &&
-      shippingCountry === "US"
+      ((processWithArtaShipping && shippingCountry === "US") ||
+        (artsyShippingInternational && shippingCountry !== "US"))
```

Comment:
the remaining Q is if we should rename artsyShippingInternational to processWithArtsyShippingInternational cc: @oxaudo @jpotts244 @ansor4

Reply:
That was not a rename suggestion. I was suggesting to add a parallel method to `processWithArtaShipping` that for now will only check for `artsyShippingInternational` in gravity. But again - don't feel too strongly about it. I'm just very skeptical that those requirements to only ever check for this one variable will not change soon....

Reply:
Thanks for further explaining! Since there were no strong opinions and we don't face a situation where we're "forcing" partners to adopt Artsy Shipping for international, I can't think of a case in the near future that will require a more complex logic than the partner just opting them in manually. I'll merge this as is

Reply:
Thanks for further explaining! Since there were no strong opinions and we don't face a situation where we're "forcing" partners to adopt Artsy Shipping for international, I can't think of a case in the near future that will require a more complex logic than the partner just opting them in manually. I'll merge this as is